app/controllers/workshops_controller.rb

```diff
@@ -46,7 +46,12 @@ def payment
     if user_signed_in? && current_user.local_school_admin?
       @resource = current_user
       @registrants = @workshop.registrants.where(local_school_id: current_user.local_school_id)
-      @amount = @workshop.price * @registrants.count
+      registrant_ids = []
+      @registrants.each do |registrant|
+        registrant_ids << registrant.id.to_s
```

Comment:
Looks like we're not using `registrant_ids` anywhere. Can we get rid of? And FYI, look at the `map` or `pluck` method. This can be done more concisely with `registrant_ids = @registrants.pluck(:id)`